### ISSUE TITLE
CSV Handler doesn't handle Big files

### ISSUE THREAD

**Issue by akhripko** [begmaroman/cick-test-go#2]
There is an issue in case when there is big CSV file. 
Try use bufio Scanner & /tmp/ folder

**Comment by begmaroman** [comment]
Good catch! I have had only 2 hours to developing 2 services and setting communication between them
